charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -125,6 +125,11 @@
 {{- end }}
 {{- end }}
 {{- end }}
+{{- if and .Values.datadog.systemProbe.enableCORE .Values.datadog.systemProbe.btfPath }}
```

Comment:
it ie enable by default, but. the btfPath is empty by default. so it means that it is disable by default.

can we just say: if btfPath is not empty we mount it?

Reply:
Done :)